This patch allows the postfix build system to accept building on Linux 4.*

Even for Linux 4.x systems, it sets SYSTYPE variable to Linux3 because
I am not sure if I'd have to change other part of the build system to
make it cope with the new Linux 4.x.  I guess I'll try in subsequent
patch to set the SYSTYPE properly and see what breaks.  For now, this
seems to work.

--- postfix-3.0.1/makedefs	2015-04-26 19:07:31.924108509 +0200
+++ postfix-3.0.1/makedefs.dodji	2015-04-26 19:06:22.678638335 +0200
@@ -500,7 +500,7 @@
 		: ${SHLIB_ENV="LD_LIBRARY_PATH=`pwd`/lib"}
 		: ${PLUGIN_LD="${CC-gcc} -shared"}
 		;;
-    Linux.3*)	SYSTYPE=LINUX3
+    Linux.3*|Linux.4*)	SYSTYPE=LINUX3
 		case "$CCARGS" in
 		 *-DNO_DB*) ;;
 		 *-DHAS_DB*) ;;
